### PR TITLE
[IMP] Inventory: new version of lead times doc

### PR DESCRIPTION
**Pull request by Lau-Odoo**

### CODE REVIEW

**Comment by Lau-Odoo** [comment]
Forgot the alt attribute and redirect.txt but you can review @jubodoo

**Comment by Lau-Odoo** [comment]
Should be all good @jubodoo

**Comment by Lau-Odoo** [comment]
Could you review this @jubodoo ?
It's a bit urgent.

File: redirects.txt [hunk 1]
```diff
@@ -129,6 +129,7 @@ inventory/management/adjustment/virtual_warehouses.rst inventory/management/repl
 inventory/settings/warehouses.rst inventory/management/warehouses.rst 																# settings/* --> management/* (#424)
 inventory/settings/warehouses/difference_warehouse_location.rst inventory/management/warehouses/difference_warehouse_location.rst   # settings/* --> management/* (#424)
 inventory/settings/warehouses/location_creation.rst inventory/management/warehouses/location_creation.rst 							# settings/* --> management/* (#424)
+inventory/management/delivery/schedule_date.rst inventory/management/planning/scheduled_dates.rst 									# delivery/schedule_date --> planning/scheduled_dates (#660)
```
Comment (AntoineVDV) [inline redirects.txt:132]: That's incorrect

Comment (Lau-Odoo) [reply redirects.txt:132]: Well seen!

**Pull request merged**